packages/atomic/src/components/facets-v1/atomic-facet/atomic-facet.tsx

```diff
@@ -119,7 +120,18 @@ export class AtomicFacet
   @Prop() public displayValuesAs: 'checkbox' | 'link' | 'box' = 'checkbox';
   // @Prop() public customSort?: string; TODO: add customSort to headless
 
+  private validateProps() {
+    new Schema({
+      displayValuesAs: new StringValue({
+        constrainTo: ['checkbox', 'link', 'box'],
+      }),
+    }).validate({
+      displayValuesAs: this.displayValuesAs,
+    });
+  }
+
   public initialize() {
+    this.validateProps();
```

Comment:
When those fails, I'm guessing it bubble up to the `quantic-component-error` component, and we display something in the console + there's a "red box" message that appears visually ?

Reply:
Yes same as when a headless option fails, the component error show and more info is available in the console